crates/core/tests/test_checkpoint.rs

```diff
@@ -0,0 +1,50 @@
+use bdk_core::CheckPoint;
+
+#[allow(unused_macros)]
+macro_rules! h {
+    ($index:literal) => {{
+        bitcoin::hashes::Hash::hash($index.as_bytes())
+    }};
+}
+
+#[allow(unused_macros)]
+macro_rules! block_id {
+    ($height:expr, $hash:literal) => {{
+        bdk_chain::BlockId {
+            height: $height,
+            hash: bitcoin::hashes::Hash::hash($hash.as_bytes()),
+        }
+    }};
+}
```

Comment:
Would it make sense to have these in a separate `mod.rs` file? If more tests are added to `bdk_core` in the future, these may be useful to have.

Reply:
Yes, I do agree, +1 on the suggestion.

Reply:
Agree. However, I think a long-term solution should be something like: https://github.com/bitcoindevkit/bdk/issues/1602

Reply:
* `common.rs` file.

Reply:
Moved `block_id` macro to `tests/common.rs`.